etl/load_mosaiks_to_db/__main__.py

```diff
@@ -17,6 +20,7 @@
 db.create_mosaiks_table()
 db.write_mosaiks_records(data)
 db.create_geo_index()
+closest_features = db.get_mosaiks_closest_features(point_lon, pont_lat, n)
```

Comment:
Will later move the use of this into a separate script so testing out `get_mosaiks_closest_features` isn't tied to the task of writing all the feature from local to DB.